pypeit/wavecalib.py

```diff
@@ -401,6 +401,7 @@ def wave_diagnostics(self, print_diag=False):
         diag['IDs_Wave_cov(%)'] = lines_cov
         diag['IDs_Wave_cov(%)'].format = '0.1f'
         # FWHM
+        # TODO: Why not me*a*sured_fwhm?
```

Comment:
yeah, this looks like a typo. It seems to have been propagated through several bits of the code and docs (e.g. `pypeit.scripts.chk_wavecalib`)

Reply:
I've changed all the instances I could find to `measured_fwhm`.